packages/commercetools/composables/src/getters/userShippingGetters.ts

```diff
@@ -0,0 +1,34 @@
+import { UserShippingGetters } from '@vue-storefront/core';
+
+export const getUserFirstName = (user: any): string => user ? user.firstName : '';
+
+export const getUserLastName = (user: any): string => user ? user.lastName : '';
+
+export const getUserFullName = (user: any): string => user ? `${user.firstName} ${user.lastName}` : '';
+
+const userGetters: UserShippingGetters<any> = {
+  getFiltered: (addresses, criteria: Record<string, any>) => {
+    return addresses.filter(address => {
+      for (const [key, value] of Object.entries(criteria)) {
+        if (address[key] !== value) {
+          return false;
+        }
+      }
+      return true;
+    });
+  },
```

Comment:
This can be simplified to:

```javascript
const entries = Object.entries(criteria);

return addresses.filter(address => entries.every(([ key, value ]) => address[key] === value))
```

Please make sure that it still works as before :)